internal/engine/image_build_and_deployer.go

```diff
@@ -102,18 +102,23 @@ func (ibd *ImageBuildAndDeployer) BuildAndDeploy(ctx context.Context, manifest m
 func (ibd *ImageBuildAndDeployer) build(ctx context.Context, manifest model.Manifest, state store.BuildState, ps *build.PipelineState) (reference.NamedTagged, error) {
 	var n reference.NamedTagged
 
-	name := manifest.DockerRef()
-	if manifest.IsStaticBuild() {
+	dInfo := manifest.DockerInfo()
+	if dInfo.Empty() {
```

Comment:
i wonder if ValidateDockerK8sManifest and this validation should be merged, so that ValidateDockerK8sManifest returns a guaranteed non-empty DockerInfo

Reply:
well this is no longer in there so mreh.